include/libsemigroups/paths.hpp

```diff
@@ -661,46 +303,46 @@ namespace libsemigroups {
   //! returned by this function are equal whenever they point to equal
   //! objects.
   //!
+  //! \param wg the WordGraph
```

Comment:
```suggestion
  //! \param wg the WordGraph.
```
(and the same for the params below)